### ISSUE TITLE
questions in complex batchnormalization and calculating E mask in the end

### ISSUE THREAD

**Issue by iris1997jiatong** [huyanxin/DeepComplexCRN#31]
Hello,

Thanks for the nice code!
I encounter NaN after 30 epochs when I train DCCRN and I try to find the problem.
I have two questions.
1. when calculating whitening matrix 
![image](https://github.com/user-attachments/assets/86536db2-56a0-43ee-b3dc-2e1532682a60)
Can you make sure that s, t and inverse_st are always available to calculate (delta>=0, tau+2s >= 0 and s*t != 0)?
2. when calculating E mask in the end, 
![image](https://github.com/user-attachments/assets/eea75f5b-1abc-4a39-8693-00d160a99f62)
Can you make sure that real phase is never equal to zero such that arctan(imag/real) is always available?
